packages/edit-site/src/components/page-pages/index.js

```diff
@@ -343,7 +345,28 @@ export default function PagePages() {
 			</Page>
 			{ view.type === LAYOUT_LIST && (
 				<Page>
```

Comment:
I think this Page component should ideally correspond to the `edit-site-layout__canvas-container` div rendered by the layout, it might take some shuffling of components to get that right.

Making these two similar means, we won't remount the component which means, we'll have a nice animation when we click this preview.

Reply:
This is not a blocker for this PR, mostly a thought for what needs to happen to make all of this stable.

Reply:
I guess in other words, that just means that we don't need the "PostPreview" component at all and when we'll make the "merge", selecting items in the dataviews will actually "navigate" (change the url) in the site editor which will automatically set the right post in the default "frame". It's going to simplify things :)